src/client/components/PageBox.tsx

```diff
@@ -8,6 +10,14 @@ export const pageBox = css`
   justify-content: start;
   align-items: start;
   width: 100%;
+  padding-top: ${space[2]}px;
+  padding-bottom: 60px;
+  ${from.mobileMedium} {
+    padding-top: ${space[4]}px;
+  }
+  ${from.tablet} {
+    padding-top: ${space[6]}px;
```

Comment:
Correctly space the page box from the header for all pages that use it, according to the responsive design rules set out in Figma